src/examples/ui/sticky-header/css/sticky-header.js

```diff
@@ -0,0 +1,162 @@
+import throttle from '../../../../assets/js/throttle.js';
+
+const debug = false;
```

Comment:
I get wanting to have this but it honestly seems overkill. I'd rather we keep things as lean as possible.

Reply:
The behavior gets complicated very quickly and I kept needing to console.log the behavior, so I baked it in. I created the `debug` setup so I could easily show anyone (like you perhaps) how the logic plays out in realtime.

It can be removed if you think its too far, but I like it for testing and troubleshooting.

Reply:
I removed all debug functionality